Dockerfile

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1.2
 
-ARG BASE_IMAGE=debian:latest
+ARG BASE_IMAGE=debian-slim:latest
```

Comment:
This is wrong. That's why the [integration test](https://github.com/bokysan/docker-postfix/actions/runs/6669595085/job/18127984095?pr=148#step:3:62) is failing.